cdap-docs/developers-manual/source/data-exploration/filesets.rst

```diff
@@ -0,0 +1,86 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _fileset-exploration:
+
+============================================
+Fileset Exploration
+============================================
+
+The ``FileSet`` and ``TimePartitionedFileSet`` datasets can be explored through ad-hoc SQL-like queries.
+To enable exploration, you must set several properties when creating the Dataset, and the files in 
+your dataset must fit certain constraints. These are described in more detail in the following sections. 
+
+Explore Properties
+------------------
+A ``FileSet`` or ``TimePartitionedFileSet`` can be made explorable by setting several properties when
```

Comment:
is made \[I think then it's clearer that it happens at the start, and not after-the-fact]